src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -82,21 +74,6 @@ const displayAttributeValue = (v: unknown): string => {
   );
```

Comment:
Should remove `displayAttributeValue` from here since you put it in `DataUseLimitations`.